component/gnap/as/client.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package as
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/trustbloc/edge-core/pkg/log"
+
+	gnaprest "github.com/trustbloc/auth/pkg/restapi/gnap"
+	"github.com/trustbloc/auth/spi/gnap"
+)
+
+// package as is the Authentication Server client that requests GNAP tokens from the Authorization Server.
```

Comment:
put this above the package clause so it's a package comment?

Reply:
done